code/android-core-library/src/main/java/com/adobe/marketing/mobile/util/TimeUtils.kt

```diff
@@ -98,7 +147,7 @@ object TimeUtils {
 
     /**
      * Converts the epoch into a RFC-2822 date string pattern - [RFC2822_DATE_PATTERN]
-     * @param epoch the epoch that should be converted to Date
+     * @param epoch the epoch (milliseconds) that should be converted to [Date]
      * @param timeZone the timezone that should be used
      * @param locale the locale whose date format symbols should be used
      * @return a RFC-2822 formatted date string for the epoch provided
```

Comment:
@praveek on getRFC2822Date - I haven't seen this one in iOS, are there any extensions using it or are we using it for logging, or can it be removed?

Reply:
This is used for the purposes of storing date from responses in cache metadata. Core 2.0 cache system uses a file backed cache metadata. So in order to verify the last modified time of the cached file from the the one in the response we store the last modified in the cache metadata. I read that the epoch is the preferred way to store date/time over a specific formatted date in file backed cache metadata and hence this logic was added.